src/main/scala/mesosphere/marathon/state/Volume.scala

```diff
@@ -364,17 +368,17 @@ object ExternalVolumeInfo {
 
   implicit val validExternalVolumeInfo = validator[ExternalVolumeInfo] { info =>
     info.size.each should be > 0L
-    info.name should matchRegex(LabelRegex)
```

Comment:
Copied from other PR: @timcharper 
instead of removing validation out-right, can we limit it to URI characters perhaps? Hesitant to allow surrounding whitespace, for example.

Reply:
I'm not sure. On one hand, this field should now contain options, which might be anything, including unicode characters for whatever I know, escaped quotes, anything really.
On the other hand, we might also think about stuff like newlines, etc....